src/tools/chip-cert/CertUtils.cpp

```diff
@@ -1112,121 +1145,126 @@ CHIP_ERROR MakeCertTLV(CertType certType, const ToolChipDN * subjectDN, X509 * c
         writer.PutBytes(ContextTag(kTag_EllipticCurvePublicKey), subjectPubkey, chip::Crypto::CHIP_CRYPTO_PUBLIC_KEY_SIZE_BYTES));
 
     // extensions
-    ReturnErrorOnFailure(writer.StartContainer(ContextTag(kTag_Extensions), kTLVType_List, containerType2));
```

Comment:
This section of the diff renders really badly, it's just wrapping this code in `if (!useCompactIdentityFormat) { ... }`

Reply:
Hiding whitespace diffs in the github UI works nicely.  ;)